upload.js

```diff
@@ -0,0 +1,33 @@
+const AWS = require("aws-sdk");
```

Comment:
I wonder if we can name this file for AWS. In the future, we may be using a different cloud provider. So we could have upload_AWS.js and upload_IBM.js, etc.